File: llm_generated_code/Gemini/CPP/Level1/Q32.cpp
#include <iostream>
#include <string>

using namespace std;

bool isMeow(const string& s) {
    int i = 0, n = s.size();
    while (i < n && (s[i] == 'm' || s[i] == 'M')) i++;
    while (i < n && (s[i] == 'e' || s[i] == 'E')) i++;
    while (i < n && (s[i] == 'o' || s[i] == 'O')) i++;
    while (i < n && (s[i] == 'w' || s[i] == 'W')) i++;
    return i == n;
}

int main() {
    int t;
    cin >> t;

    while (t--) {
        string s;
        cin >> s;

        if (isMeow(s)) {
            cout << "YES" << endl;
        } else {
            cout << "NO" << endl;
        }
    }

    return 0;
}

File: llm_generated_code/Gemini/Python/Level1/Q32.py
def is_meow(s):
    i = 0
    while i < len(s) and (s[i] == 'm' or s[i] == 'M'):
        i += 1
    while i < len(s) and (s[i] == 'e' or s[i] == 'E'):
        i += 1
    while i < len(s) and (s[i] == 'o' or s[i] == 'O'):
        i += 1
    while i < len(s) and (s[i] == 'w' or s[i] == 'W'):
        i += 1
    return i == len(s)

def main():
    t = int(input())
    for _ in range(t):
        s = input()
        print("YES" if is_meow(s) else "NO")

if __name__ == "__main__":
    main()
